extensions/oplogPopulator/OplogPopulator.js

```diff
@@ -6,12 +6,15 @@ const { constructConnectionString } = require('../utils/MongoUtils');
 const ChangeStream = require('../../lib/wrappers/ChangeStream');
 const Allocator = require('./modules/Allocator');
 const ConnectorsManager = require('./modules/ConnectorsManager');
+const { ZenkoMetrics } = require('arsenal').metrics;
+const OplogPopulatorMetrics = require('./OplogPopulatorMetrics');
 
 const paramsJoi = joi.object({
     config: joi.object().required(),
     mongoConfig: joi.object().required(),
     activeExtensions: joi.array().required(),
     logger: joi.object().required(),
+    enableMetrics: joi.boolean().default(true),
```

Comment:
do we need to make metrics optional?
i don't really see a use-case here, metrics should always be collected?

Reply:
In normal cases we always have the metrics enabled, that's why it's set to true by default.
The option was put there to be able to run unit tests with no issues, as otherwise we would get errors about the metrics being registered multiple times.

Reply:
we may indeed have something for the tests, but it should not propagate to the config...